### DISCUSSION TITLE
How can I bind data to a style?

### DISCUSSION THREAD

**Discussion by aleksb3** [webstudio-is/webstudio-community#145] [Q&A] [answered]
I'm trying to extract a value from my directus CMS instance that's a colour code (ie. #EFEFEF) and apply this value to the background of an image that is part of a collection.

Is it possible to apply extracted data from a CMS and apply it to a style like this?

**Comment by kof** [comment, marked as answer]
Bindings for styles are not yet supported yet, only properties panel and page settings, follow this issue https://github.com/webstudio-is/webstudio/issues/3416
